python/docs/source/index.rst

```diff
@@ -1,49 +1,32 @@
-Vowpal Wabbit Python Wrapper
-============================
+Vowpal Wabbit Python Wrapper Documentation
+==========================================
 
-Vowpal Wabbit is a fast machine learning library for online learning, and this is the python wrapper for the project.
+The Vowpal Wabbit (VW) project is a fast out-of-core learning system sponsored by Microsoft Research and (previously) Yahoo! Research.
 
 
-Code Documenation
------------------
+Vowpal Wabbit Documentation Websites
+------------------------------------
+
 
-See documenation for the following modules in the package:
 
 .. toctree::
-   vowpalwabbit.pyvw
-   vowpalwabbit.sklearn
-
-Usage
------
-
-You can use the python wrapper directly like this:
-
-.. code-block:: python
-
-    from vowpalwabbit import pyvw
-    vw = pyvw.vw(quiet=True)
-    ex = vw.example('1 | a b c')
-    vw.learn(ex)
-    vw.predict(ex)
-
-Or you can use the included scikit-learn interface like this:
-
-.. code-block:: python
-
-    import numpy as np
-    from sklearn import datasets
-    from sklearn.model_selection import train_test_split
-    from vowpalwabbit.sklearn_vw import VWClassifier
-        # generate some data
-    X, y = datasets.make_hastie_10_2(n_samples=10000, random_state=1)
-    X = X.astype(np.float32)
-        # split train and test set
-    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=256)
-        # build model
-    model = VWClassifier()
-    model.fit(X_train, y_train)
-        # predict model
-    y_pred = model.predict(X_test)
-        # evaluate model
-    model.score(X_train, y_train)
-    model.score(X_test, y_test)
+   :maxdepth: 1
+
+   python_wrapper/index
```

Comment:
Why push this to a deeper page?

Reply:
1-Later maybe we will have more modules in python directory so I am including this in a single doc and then for the newly implemented module we can easily add a new folder and can include it into the `index.rst`.
2- I am thinking to add general information/details of the vowpal wabbit on the index page.

Reply:
Also, I will add other sources of doc and tutorials for vowpal wabbit. It can be helpful as doc will be connected.